common/pulp_docker/common/error_codes.py

```diff
@@ -15,7 +15,7 @@
     "lower case letters, integers, hyphens, periods, and may include a single slash."),
     ['field', 'value'])
 DKR1006 = Error("DKR1006", _(
-    "If %(field)s is not specified, it will default to the id must contain only lower case letters,"
-    " integers, hyphens, periods, and may include a single slash. Please specify a valid registry "
-    "id or change the repo id."),
+    "%(field)s may only contain lower case letters, integers, hyphens, periods, and may include "
+    "a single slash. If not specified, it will default to the repo id which did not meet "
+    "these requirements."),
```

Comment:
How about changing the second sentence to something like this?

"When %(field)s is not specified, the repo-id value is used. In that case the repo-id needs to adhere to the same requirements as %(field)s."